grammar.js

```diff
@@ -572,7 +596,7 @@ module.exports = grammar({
         )
       ),
 
-    command_name: $ => /\\([^\r\n]|[@a-zA-Z:_]+\*?)?/,
+    command_name: $ => /\\([^\r\n]|[@a-zA-Z]+\*?)?/,
```

Comment:
Removing `:` and `_` here breaks recognizing `LaTeX3` commands (https://mirror.dogado.de/tex-archive/macros/latex/contrib/l3kernel/expl3.pdf) but I think we should add a dedicated rule for `LaTeX3` commands instead, which follows the syntax defined in the PDF more closely.

Reply:
ahh, this is the first time I'm learning about `Latex3` haha. I'd be happy to work on this if you'd like.

Reply:
> I'd be happy to work on this if you'd like.

Sure :+1: